keep-only-warm-data/docker/push_images.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo "Push to Docker Hub"
+docker push kuzzleio/howto-keepwarmdata-kuzzle
```

Comment:
Why not rely on docker hub automatic builds?

Reply:
Travis need the images to run the test, if I push a commit that need the latest version of images to be tested, Docker Hub will not be updated at time.

Reply:
Travis will build and push the images before running the tests

Reply:
I will do this in a different PR.